.gitlab/build.yml

```
@@ -84,8 +61,10 @@ build_x64:
         IMAGE: kernel-version-testing_x64
       - DOCKERFILE: agent-deploy/Dockerfile
         IMAGE: gitlab_agent_deploy
-      - DOCKERFILE: linux-glibc-2.17-x64/Dockerfile
-        IMAGE: linux-glibc-2-17-x64
+      - DOCKERFILE: linux-glibc/Dockerfile
+        IMAGE: linux-glibc-x64
```

> There is no more the glibc version in the name of the image.

Why not 
```suggestion
        IMAGE: linux-glibc-2-17-x64
```
then?

Or otherwise, if we want to deemphasize the glibc aspect of it, we may as well drop `glibc` from the name and name it after its main expected use, something like:
```suggestion
        IMAGE: linux-agent-build-x64
```

---

Definitely agree with the second suggestion, especially since the plan we discussed with @chouetz is to further merge all buildimages used in CI into one, i.e. "the one image ~~to rule them all~~ to build the whole agent". I don't see why we would emphasize the glibc aspect of it.